install/install.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# -*- ENCODING: UTF-8 -*-
+
+## Antiguo script de instalación: https://gist.github.com/cristiannsc/465a9899d28c3093821412a29ded834e
+
+## Update the ubuntu/debian box
+echo "Installing PHP...\n"
+sudo apt-get update && apt-get upgrade
+sudo apt-get install nginx php7.2-fpm php7.2-common php-memcache\
+      php7.2-mbstring php7.2-xmlrpc php7.2-gd php7.2-xml php7.2-cgi \
+      php7.2-mysql php7.2-cli php7.2-zip php7.2-curl php7.2-bcmath -y
+sudo apt-get -y install gcc make autoconf libc-dev pkg-config
+sudo apt-get -y install php7.2-dev
+sudo apt-get -y install libmcrypt-dev
+sudo pecl install mcrypt-1.0.1
+
+echo "Installing MySQL...\n"
+sudo debconf-set-selections <<< 'mysql-server-5.7 mysql-server/root_password password rootpass'
+sudo debconf-set-selections <<< 'mysql-server-5.7 mysql-server/root_password_again password rootpass'
+sudo apt-get install mysql-server-5.7 python-mysqldb -y
+
+echo "Getting minimal nginx configuration file\n"
+wget -O /etc/nginx/sites-available/default \
+    https://gist.githubusercontent.com/cristiannsc/1d4ae9267f433a7927076c8bd79ff9e3/raw/default
+
+wget -O /etc/nginx/php_fastcgi \
+    https://gist.githubusercontent.com/cristiannsc/36c765c85daff7364e2c87639af399da/raw/php_fastcgi
```

Comment:
Who is this christiannsc and why all those wget commands to fetch his random gists?